### DISCUSSION TITLE
What’s the best way to create a CLI Application for Radiology Inference?

### DISCUSSION THREAD

**Discussion by soumen02** [Project-MONAI/MONAILabel#1701] [Q&A]
Hello all,
I have a radiology app that I would like to package into an application that I can run using command-line (with --input and --output arguments or something similar). Is there a streamlined way to do this? I don’t wish to start server or use the default monailabel datastore.
One way I am doing this right now is using the main.py under radiology, but this includes other functions like training, scoring etc. So I’m searching for a way to only have inference using a pre-trained model - maybe something similar to the TotalSegmentator App that we can install using pip. 
Thanks!

**Comment by diazandr3s** [comment]
Hi @soumen02,

If you only need the inference part, there are many changes to be made if you want to remove the datastore, REST API, etc.
Have you considered using/training the Auto3DSeg? Here we show an example of how to run inference using the CLI: https://github.com/lassoan/SlicerMONAIAuto3DSeg/issues/52#issuecomment-2075577774

Hope this helps,
